### PR TITLE
Create caliptra.md

### PR DESCRIPTION
**Pull request by mcockrell-google**

### CODE REVIEW

File: projects/sandbox/caliptra.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Application
+
+### Sandbox application
+
+* Project name: Caliptra
+* Project repo(s):
+
+| Repo | What |
+| ---- | ---- |
+| [Primary Repo](https://github.com/chipsalliance/caliptra)        | This repo, with admin boilerplate and docs |
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:12]: s/This/Base non-code/

File: projects/sandbox/caliptra.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| [Caliptra RTL](https://github.com/chipsalliance/caliptra-rtl)    | The primary repository with immutable RTL |
+| [Caliptra SW](https://github.com/chipsalliance/caliptra-sw)      | Caliptra software (ROM, FMC, runtime firmware), and libraries/tools needed to build and test |
+| [Caliptra U-Reg](https://github.com/chipsalliance/caliptra-ureg) | General-purpose libraries and tools for manipulating MMIO registers from Rust |
+| [Caliptra DPE](https://github.com/chipsalliance/caliptra-dpe)    | An implementation of a TCG DICE Protection Environment profile |
+* Brief summary of the project: The Caliptra project focuses on development of HW and SW IP for [The Caliptra Root of Trust](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md).
+* Project's open source license: **ASL 2.0**
+* Link to issue tracker: **TBD**
+* Link to website: **TBD**
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:20]: caliptra.io

File: projects/sandbox/caliptra.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| Repo | What |
+| ---- | ---- |
+| [Primary Repo](https://github.com/chipsalliance/caliptra)        | This repo, with admin boilerplate and docs |
+| [Caliptra RTL](https://github.com/chipsalliance/caliptra-rtl)    | The primary repository with immutable RTL |
+| [Caliptra SW](https://github.com/chipsalliance/caliptra-sw)      | Caliptra software (ROM, FMC, runtime firmware), and libraries/tools needed to build and test |
+| [Caliptra U-Reg](https://github.com/chipsalliance/caliptra-ureg) | General-purpose libraries and tools for manipulating MMIO registers from Rust |
+| [Caliptra DPE](https://github.com/chipsalliance/caliptra-dpe)    | An implementation of a TCG DICE Protection Environment profile |
+* Brief summary of the project: The Caliptra project focuses on development of HW and SW IP for [The Caliptra Root of Trust](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md).
+* Project's open source license: **ASL 2.0**
+* Link to issue tracker: **TBD**
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:19]: Should we tweak the main README.md and/or this .md with link to each issue tracker per repo?

File: projects/sandbox/caliptra.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  * Role within the project: IC
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
+* Link to logo in .svg format
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:37]: Yeah I don't have that. Is this a hard need?

File: projects/sandbox/caliptra.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance:
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance:
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:40]: Will do PR for this later today

File: projects/sandbox/caliptra.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
+* Link to logo in .svg format
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:38]: Just say yes :)

File: projects/sandbox/caliptra.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  * GitHub handle: mcockrell-google
+  * Role within the project: IC
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:36]: spec.md?

File: projects/sandbox/caliptra.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+* Link to logo in .svg format
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance:
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance:
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance:
+* Confirmation that the contributor will assign or transfer any common law or registered trademark rights, project accounts (e.g. GitHub), and project domain names to the Linux Foundation upon acceptance:
+* Link to documented process for reporting security vulnerabilities:
+* **For specifications** Confirmation there is at least one public reference implementation:
+* Primary contact from the project during the Graduation process:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:44]: That could be me

File: projects/sandbox/caliptra.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
+* Link to logo in .svg format
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance:
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance:
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance:
+* Confirmation that the contributor will assign or transfer any common law or registered trademark rights, project accounts (e.g. GitHub), and project domain names to the Linux Foundation upon acceptance:
+* Link to documented process for reporting security vulnerabilities:
+* **For specifications** Confirmation there is at least one public reference implementation:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:43]: NA?

File: projects/sandbox/caliptra.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
+* Link to draft mission statement:
+* Link to logo in .svg format
+* Confirmation that the project adopts the [CHIPS Alliance Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) upon acceptance:
+* Confirmation that the project will adopt the [CHIPS Alliance IP policy](https://technical-charter.chipsalliance.org) upon acceptance:
+* Confirmation that the project will add CHIPS Alliance header or footer text and links to its websites upon acceptance:
+* Confirmation that the contributor will assign or transfer any common law or registered trademark rights, project accounts (e.g. GitHub), and project domain names to the Linux Foundation upon acceptance:
+* Link to documented process for reporting security vulnerabilities:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:42]: Prabhu from AMD says CHIPS themselves does not have PSIRT. Skip?

File: projects/sandbox/caliptra.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+  * Email: mcockrell@google.com
+  * GitHub handle: mcockrell-google
+  * Role within the project: IC
+
+*Projects applying for Sandbox status may stop here.*
+
+### Graduation application
+
+* Total number of active committers:
+* Brief description of release methodology and mechanics:
```
Comment (andreslagarcavilla) [inline projects/sandbox/caliptra.md:35]: CONTRIBUTING.md (when merged)

**Comment by cjyabraham** [comment]
The new yml format can be seen here: [f4pga.yml](https://github.com/chipsalliance/tsc/blob/main/projects/f4pga.yml)
And project logo can be placed in [tsc/projects/logos/](https://github.com/chipsalliance/tsc/tree/main/projects/logos)

**Comment by andreslagarcavilla** [comment]
we have pngs, let me dig svg

**Comment by mgielda** [comment]
Can you put the below in a caliptra.yml (not md) file for now? Of course you can change any of this. For repos, we don't support descriptions for now (I think) but it's a fixable problem.

```# Project name
name: Caliptra
# Project status - sandbox, graduated or archived
status: sandbox
# Links to project repos - may be one or list of several in YAML list format
repositories:
  - https://github.com/chipsalliance/caliptra
  - https://github.com/chipsalliance/caliptra-rtl
  - https://github.com/chipsalliance/caliptra-sw
  - https://github.com/chipsalliance/caliptra-ureg
  - https://github.com/chipsalliance/caliptra-dpe
# A one-sentence summary of the project
brief_summary: The Caliptra project focuses on development of HW and SW IP for The Caliptra Root of Trust
# The project's open source license, normally Apache 2.0 unless special exception is granted
license: Apache-2.0
# Link to issue tracker
issue_tracker_link: https://github.com/chipsalliance/Caliptra/issues
# Link to website
website_link: https://github.com/chipsalliance/Caliptra/blob/main/README.md
# Links to social media, if present, can be a list, or N/A if none
social_media_links: N/A
# Brief summary of how widely the project is used e.g. list of companies
project_usage_and_scale: Google, AMD, Microsoft, NVIDIA and other project particupants. 
# Project's motivation to join CHIPS
why_join_chips: CHIPS Alliance is a common forum for Caliptra development enabling enable cross-company collaboration, adoption of CHIPS open source RTL tools and collaborative methodologies and outreach to a broader community of interested parties
# The primary contact person for the project, will become TSC representative once admitted
primary_contact:
  # Contact's full name
  name: Matt Cockrell
  # Contact's work email
  email: mcockrell@google.com
  # Contact's GH handle, without the @
  github_handle: mcockrell-google
  # Contact's role in the project
  project_role: IC
### Data above this line is needed for Sandbox application, additional data below is needed for Graduated application
# Link to logo in .svg format, if present
svg_logo_link: logos/caliptra.svg
```

We can fill in the remaining data as needed later, when we graduate the project.

**Comment by mgielda** [comment]
Ah, and please add a logo to projects/logos/ in the same PR. I saw the logo is already there in the Caliptra repo.

**Comment by mcockrell-google** [comment]
@mgielda - I don't see any .yml files in the projects/graduated. Am I looking in the wrong place?

File: projects/sandbox/caliptra.md (hunk 3 above)
Comment (mcockrell-google) [reply projects/sandbox/caliptra.md:19]: I would say the main README, seems cleaner. Can I just put a link to the README?

File: projects/sandbox/caliptra.md (hunk 4 above)
Comment (mcockrell-google) [reply projects/sandbox/caliptra.md:37]: We will use png

File: projects/caliptra.yml [hunk 12]
```diff
@@ -0,0 +1,37 @@
+name: Caliptra
+# Project status - sandbox, graduated or archived
+status: sandbox
+# Links to project repos - may be one or list of several in YAML list format
+repositories:
+  - https://github.com/chipsalliance/caliptra
+  - https://github.com/chipsalliance/caliptra-rtl
+  - https://github.com/chipsalliance/caliptra-sw
+  - https://github.com/chipsalliance/caliptra-ureg
+  - https://github.com/chipsalliance/caliptra-dpe
```
[the remark this answers is not in the document]
Comment (mcockrell-google) [reply projects/caliptra.yml:10]: Can I go ahead and merge?

**Pull request merged**